src/module/party/party.js

```diff
@@ -5,20 +5,13 @@ export class OseParty {
    * @return {OseActor[]}
    */
   static get currentParty() {
-    const v10 = isNewerVersion(game.version, "10.264");
-    const systemName = game.system.id == 'ose' ? game.system.id : 'ose-dev'
-    const characters = v10 ?
-      game.actors.filter(
-        (act) =>
-          act.type === "character" &&
-          act.flags[systemName] &&
-          act.flags[systemName].party === true) :
-      game.actors.filter(
-        (act) =>
-          act.data.type === "character" &&
-          act.data.flags[systemName] &&
-          act.data.flags[systemName].party === true);
-    console.log(characters)
+    const systemName = game.system.id == "ose" ? game.system.id : "ose-dev";
```

Comment:
Can this come from `CONFIG.OSE`?

Reply:
addressed in third commit today named "fix: one-use keys..."